### ISSUE TITLE
libselinux-python with virtualenvs require manual copy

### ISSUE THREAD

**Issue by termlen0** [ansible/workshops#205]
##### SUMMARY
If selinux is enabled on the provisioning host and a virutal environment is being used to execute the playbook the operator may run into the following error (Assuming the libselinux-python library was installed using yum)

```
TASK [manage_ec2_instances : save private key] *********************************************************************
fatal: [localhost]: FAILED! => changed=false 
  checksum: c507856105443cc8e2df2a8bb385c63fb170977f
  msg: Aborting, target uses selinux but python bindings (libselinux-python) aren't installed!
```

The workarounds are described here: https://github.com/trailofbits/algo/issues/356
It might make sense to add this info into our FAQs as well.

##### ISSUE TYPE
- Documentation Report

**Comment by Chrislevi** [comment]
I'm having the same issue on:
Centos7
ansible 2.7.6
python3.6

Tried using `--system-site-packages` though it didnt bypassed the `libselinux-python` error. From what i've digged, my belly guess is that I do have the selinux @ `python2.7` site-packages.. but not the same for `python3.6` site-packages. so I guess `--system-site-packages` is pointed at `python2.7` by default.

I did try installing  `libselinux-python` on remote and localhost, my failing module is template.
